numerics/fma_test.cpp

```diff
@@ -18,8 +18,7 @@ class FMATest : public testing::Test {};
 TEST_F(FMATest, FMA) {
   // Note that we test even if |UseHardwareFMA| is false, i.e., even in debug.
   if (!CanEmitFMAInstructions || !HasCPUFeatures(CPUFeatureFlags::FMA)) {
```

Comment:
It would probable make sense to put that code in the `SetUp` method of the fixture, since I don't imagine any test in this file ever wanting to run if there is no FMA.